sys/dev/ispfw/asm_2500.h

```diff
@@ -16079,6 +16079,6 @@ static const uint32_t isp_2500_risc_code[] = {
 	0x00000000, 0x00000000, 0x00000000, 0x00000000,
 	0x00000000, 0x00000000, 0x00000000, 0x00000000,
 	0x00000000, 0x00000000, 0x000f1f01, 0xc3002003,
-	0xb9f6ad13, 0xffffb7ea
+	0xb9f6ad13, 0xffffb7ea, 0xffee7e37
```

Comment:
What is this change?

Reply:
Nice catch - this actually belongs to the previous commit. Fixed this by rebasing.
Background:
The firmware of those adapters is part of a so called MBI (MultiBootImage) that consists of many parts.
In a first attempt I've extracted the firmware "manually" by finding the starting point and doing some calculations to find the end. This was fine so far, but bad for new releases of the MBI/firmware in the future.
After I finished all the other parts of isp(4) and the pull request was out, I wrote a small python script to automatically extract the firmware part from the MBI.
While comparing the manually and automatically extracted firmware I found that there is a one DWORD difference in the firmware for the 25xx adapters.
I triple checked everything and I'm now sure that I made an off by one error in my first calculations and manually extracted one DWORD less out of the MBI.
So this was just to fix my calculation error and is now included in the previous firmware commit.